tests/score_variants/test_category_score.py

```diff
@@ -0,0 +1,242 @@
+                           'plugin_a')
+    config_parser: ConfigParser = ConfigParser(config_file.filename)
+    # WHEN scoring a variant
+    score, min, max = get_category_score(variant=variant_0,
+                                         category='CategoryA',
+                                         config_parser=config_parser)
+    # THEN expect proper score and min max bounds
+    assert score == expected[0]
+    assert min == expected[1]
+    assert max == expected[2]
+
+
+@pytest.mark.parametrize('aggregation_mode,expected',
+                         [('sum', (3, 0, 3)),
+                          ('max', (2, 0, 2)),
+                          ('min', (1, 0, 1))])
+def test_single_category_two_plugin(config_file, variant_0, aggregation_mode, expected):
+    """
+    Test two categories.
+    """
+    # GIVEN a scoring config in mode aggregation_mode
+    config_file.add_category('CategoryA', aggregation_mode)
+    config_file.add_plugin('CategoryA',
+                           'PluginA',
+                           {'rare': {'score': 2.0, 'lower': 0.01, 'upper': 0.05}},
+                           'plugin_a')
+    config_file.add_plugin('CategoryA',
+                           'PluginB',
+                           {'rare': {'score': 1.0, 'lower': 4.0, 'upper': 6.0}},
+                           'plugin_b')
+    config_parser: ConfigParser = ConfigParser(config_file.filename)
+    # WHEN scoring a variant
+    score, min, max = get_category_score(variant=variant_0,
+                                         category='CategoryA',
+                                         config_parser=config_parser)
+    # THEN expect proper score and min max bounds
+    assert score == expected[0]
+    assert min == expected[1]
+    assert max == expected[2]
+
+
+@pytest.mark.parametrize('aggregation_mode,expected',
+                         [('sum', (2, 0, 2)),
+                          ('max', (2, 0, 2)),
+                          ('min', (2, 0, 2))])
+def test_multi_category_two_plugins(config_file, variant_0, aggregation_mode, expected):
+    """
+    Test with a single category, shall ignore other category
+    """
+    # GIVEN a scoring config in mode aggregation_mode
+    config_file.add_category('CategoryA', aggregation_mode)
+    config_file.add_category('CategoryB', aggregation_mode)
+    config_file.add_plugin('CategoryA',
+                           'PluginA',
+                           {'rare': {'score': 2.0, 'lower': 0.01, 'upper': 0.05}},
+                           'plugin_a')
+    config_file.add_plugin('CategoryB',
+                           'PluginB',
+                           {'rare': {'score': 1.0, 'lower': 4.0, 'upper': 6.0}},
+                           'plugin_b')
+    config_parser: ConfigParser = ConfigParser(config_file.filename)
+    # WHEN scoring a variant
+    score, min, max = get_category_score(variant=variant_0,
+                                         category='CategoryA',
```

Comment:
Hehe, I totally misread this at first, expecting the min and max for the entire score above (both categories taken together). 😄  Perhaps clarify that in the comment for sloppy readers like me.